I3uckwheat/cards-of-carousal
# Bug/279 selecting left players cards causes crash

#### 0. Make sure your issue is linked:

closes: #279 

#### 1. Purpose:

This addition, removes the disconnected/kicked player's card from the czar's submittedCards array.


#### 2. Implementation:

The solution was pretty simple, we had just forgotten to send a message to the other players (mainly czar) to remove the disconnected players card. 


#### 3. Possible Issues:

This should work properly both when a player disconnects OR when the player is kicked. I tested both but it wouldn't hurt to check again.

#### 4. How to Test:

Players card is removed from czar's selection
- run frontend and backend
- open four tabs
- host in the first tab
- join in the other three tabs
- on the hosts screen, start the game
- on the player screens, submit cards
- close tab (or kick) one of the players (not czar) 
- the disconnected player's card should be removed from czar selection

Host sees pregame screen when to many players have left
- run frontend and backend
- open three tabs
- host on the first tab
- join on the other two
- on hosts screen, start the game
- close one of the player tabs
- host should show pregame screen

#### 5. Cleanup
